Incompatibility with Thorium

**Describe the bug**
When Blanket and [Thorium](https://www.curseforge.com/minecraft/mc-mods/thorium), a mod containing a collection of bugfixes, are installed together, the launcher crashes before the game window even appears.

**To Reproduce**
Steps to reproduce the behavior:
1. Download and install [Thorium](https://www.curseforge.com/minecraft/mc-mods/thorium/files/3692154)
2. Install Blanket
3. Start the game through the launcher

**Expected behavior**
Crashes in the middle of the launching process.

**Desktop (please complete the following information):**
 - Minecraft 1.18.2
 - Fabric API 0.48.0
 - Blanket 1.0.1
 - Thorium 1.0.0

**Minecraft crash report**
```
[main/ERROR]: Mixin apply for mod thorium failed thorium.mixins.json:server.WolfEntityMixin -> net.minecraft.class_1493: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException InjectionPoint(Shift)[@At("INVOKE")] on net/minecraft/class_1493::fixedGetTailAngleWhenNotAngry with priority 1000 cannot inject into net/minecraft/class_1493::method_6714()F merged by io.github.blanketmc.blanket.mixin.fixes.WolfEntity_tailSpinMixin with priority 1000 [PREINJECT Applicator Phase -> thorium.mixins.json:server.WolfEntityMixin -> Prepare Injections ->  -> handler$zji000$fixedGetTailAngleWhenNotAngry(Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;)V -> Prepare]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: InjectionPoint(Shift)[@At("INVOKE")] on net/minecraft/class_1493::fixedGetTailAngleWhenNotAngry with priority 1000 cannot inject into net/minecraft/class_1493::method_6714()F merged by io.github.blanketmc.blanket.mixin.fixes.WolfEntity_tailSpinMixin with priority 1000 [PREINJECT Applicator Phase -> thorium.mixins.json:server.WolfEntityMixin -> Prepare Injections ->  -> handler$zji000$fixedGetTailAngleWhenNotAngry(Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;)V -> Prepare]
	at org.spongepowered.asm.mixin.injection.code.Injector.findTargetNodes(Injector.java:305) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.injection.code.Injector.find(Injector.java:240) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.prepare(InjectionInfo.java:421) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1338) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1043) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:393) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:269) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:83) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:86) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) [?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:85) [client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:291) [client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:153) [client-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.3.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
[21:41:42] [main/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1493 failed
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:418) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.3.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
Caused by: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1493 failed
	at net.minecraft.class_1299.<clinit>(class_1299.java:269) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:83) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:86) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:85) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:291) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:153) ~[minecraft-1.18.2-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) ~[fabric-loader-0.13.3.jar:?]
	... 10 more
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1493 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:252) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:269) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:83) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:86) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:85) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:291) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:153) ~[minecraft-1.18.2-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) ~[fabric-loader-0.13.3.jar:?]
	... 10 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:269) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:83) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:86) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:85) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:291) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:153) ~[minecraft-1.18.2-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) ~[fabric-loader-0.13.3.jar:?]
	... 10 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [thorium.mixins.json:server.WolfEntityMixin] from phase [DEFAULT] in config [thorium.mixins.json] from mod [thorium] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:589) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:269) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:83) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:86) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:85) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:291) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:153) ~[minecraft-1.18.2-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) ~[fabric-loader-0.13.3.jar:?]
	... 10 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: InjectionPoint(Shift)[@At("INVOKE")] on net/minecraft/class_1493::fixedGetTailAngleWhenNotAngry with priority 1000 cannot inject into net/minecraft/class_1493::method_6714()F merged by io.github.blanketmc.blanket.mixin.fixes.WolfEntity_tailSpinMixin with priority 1000 [PREINJECT Applicator Phase -> thorium.mixins.json:server.WolfEntityMixin -> Prepare Injections ->  -> handler$zji000$fixedGetTailAngleWhenNotAngry(Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;)V -> Prepare]
	at org.spongepowered.asm.mixin.injection.code.Injector.findTargetNodes(Injector.java:305) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.injection.code.Injector.find(Injector.java:240) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.prepare(InjectionInfo.java:421) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1338) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1043) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:393) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.2+mixin.0.8.5.jar:0.11.2+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:269) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:83) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:86) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:85) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:291) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:153) ~[minecraft-1.18.2-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) ~[fabric-loader-0.13.3.jar:?]
	... 10 more
```


**Additional context**
I only installed Fabric API, Blanket and Thorium through MultiMC.

--- a/src/main/java/io/github/blanketmc/blanket/mixin/fixes/WolfEntity_tailSpinMixin.java
+++ b/src/main/java/io/github/blanketmc/blanket/mixin/fixes/WolfEntity_tailSpinMixin.java
@@ -7,7 +7,9 @@ import net.minecraft.entity.passive.TameableEntity;
 import net.minecraft.entity.passive.WolfEntity;
 import net.minecraft.world.World;
 import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 
 @Mixin(WolfEntity.class)
 public abstract class WolfEntity_tailSpinMixin extends TameableEntity implements Angerable {
@@ -19,18 +21,13 @@ public abstract class WolfEntity_tailSpinMixin extends TameableEntity implements
 
     /**
      * @author FX - PR0CESS
+     * @author KosmX
      * @reason Fix for the wolf's tail spinning
      */
-    @Overwrite
-    public float getTailAngle() {
-        if (this.hasAngerTime()) {
-            return 1.5393804F;
-        } else {
-            if (this.isTamed()) {
-                if (Config.wolfTailSpinFix) return (0.15F + 0.4F * (this.getHealth() / this.getMaxHealth())) * 3.1415927F;
-                return (0.55F - (this.getMaxHealth() - this.getHealth()) * 0.02F) * 3.1415927F;
-            }
-            return 0.62831855F;
+    @Inject(method = "getTailAngle", at = @At(value = "RETURN", ordinal = 1), cancellable = true)
+    public void getTailAngle(CallbackInfoReturnable<Float> cir) {
+        if (Config.wolfTailSpinFix) {
+            cir.setReturnValue ((0.15F + 0.4F * (this.getHealth() / this.getMaxHealth())) * 3.1415927F);
         }
     }
 

